delivery_postlogistics/wizard/stock_picking_return.py

```diff
@@ -0,0 +1,12 @@
+from odoo import models
+
+
+class ReturnPicking(models.TransientModel):
+    _inherit = "stock.return.picking"
+
+    def _create_returns(self):
+        new_picking, pick_type_id = super(ReturnPicking, self)._create_returns()
+        if self.picking_id.delivery_type == "postlogistics":
+            picking = self.env["stock.picking"].browse(new_picking)
```

Comment:
so new_picking is an ID in fact?

Reply:
fair point, could be named `new_picking_id`
but we tried to be consistent with upstream :) https://github.com/odoo/odoo/blob/16.0/addons/delivery/models/stock_picking.py#L329